<?xml version="1.0"?>

<launch>

  <!-- Launch the filter of kalman node -->
  <node pkg="filter_kalman" type="filter_kalman" name="filter_kalman" output="screen" >
    <param name="map_length"        type="double"   value="200"          /><!--40-->
    <param name="map_resolution"    type="double"   value="0.5"         /><!--1-->
    <param name="filter_radius"     type="double"   value="0.13"         />
    <param name="height_compensate" type="double"   value="0"		/><!--1.7-->
    <param name="fusedMapLength"    type="double"   value="200"	/>
    <param name="point_cloud_type"    type="double"   value="2.0"	/><!-- 1.0: sensor_msgs::PointCloud   2.0:sensor_msgs::PointCloud2-->
<!--useless for this project, just ignore it -->
    <param name="position1"	    type="double"   value="4"	/><!--positionx测试一下中间点滤波，此参数如需使用，请删除-->
    <param name="position2"	    type="double"   value="1.5"	/><!--positiony-->
<!--For coordinate system-->
    <param name="sub_points"    type="string"   value="/velodyne_points"  /><!--1:/velodyne16RingPC  2:/kitti/velo/pointcloud-->
    <param name="SensorFrame"   type="string"   value="/uav"       /><!-- robotic coordinate system-->
   
    <param name="FixFrame"      type="string"   value="/world"/><!-- fixed coordinate system-->

    <param name="use_forget"	type="bool"    	value="false"/>
    <param name="use_KF"	type="bool"	value="false"/>
    <param name="use_ESMF"	type="bool"	value="true"/>
    <param name="use_FilterMap"	type="bool"	value="false"			/>

    <param name="use_analyse_ESMF"	type="bool"	value="false"			/>



  </node>
</launch>
